polaris-react/src/components/ResourceList/ResourceList.tsx

```diff
@@ -48,10 +48,11 @@ function getAllItemsOnPage<TItemType>(
   });
 }
 
-const isSmallScreen = () => {
+const isBreakpointsXS = () => {
   return typeof window === 'undefined'
     ? false
-    : window.innerWidth < SMALL_SCREEN_WIDTH;
+    : window.innerWidth <
+        parseFloat(toPx(tokens.breakpoints['breakpoints-sm'].value) ?? '');
 };
```

Comment:
Same as the above: https://github.com/Shopify/polaris/pull/6364/files#r905438129